it/JmsRequestReplyClientISpec.scala

```diff
@@ -38,18 +43,18 @@ class JmsRequestReplyClientISpec extends FixtureAsyncFreeSpec with AsyncIOSpec w
 
   implicit val logger: SelfAwareStructuredLogger[IO] = Slf4jFactory[IO].getLogger
 
-  private val serviceId = "1234"
+  private val serviceId = ResponseBuilder.sid1
   private val requestQueueName = "request-general"
   private val replyQueueName = "omega-editorial-web-application-instance-1"
   private val messagingServerHost = "localhost"
   private val messagingServerPort = 9324
 
-  private val echoServer = new EchoServer
+  private val stubServer = new StubServer
 
   override def beforeAll(): Unit =
-    echoServer.startEchoServer.unsafeToFuture().onComplete {
+    stubServer.startStubServer.unsafeToFuture().onComplete {
```

Comment:
I introduced this, but now I'm thinking that the `onComplete` provides no value, as it never completes while the test is running.

Reply:
Makes sense.

This block here is a little tricky because the stub server isn't necessarily started before the first test is run, but I don't think that's a problem as the requests seem to be queued up before a connection to the JMS server is made. I think it works for now.